clj-refactor.el

```diff
@@ -2199,6 +2200,12 @@ the alias in the project."
           (-when-let (long (cljr--aget cljr-magic-require-namespaces short))
             (list short (list long))))))))
 
+(defun cljr--in-sans-ns-keyword-p ()
```

Comment:
Fine by me. I was merely pointing out a related observation.

On 25 March 2016 at 20:49, Benedek Fazekas <notifications@github.com> wrote:

> In clj-refactor.el
> <https://github.com/clojure-emacs/clj-refactor.el/pull/311#discussion_r57476657>
> :
>
> > @@ -2199,6 +2200,12 @@ the alias in the project."
> >            (-when-let (long (cljr--aget cljr-magic-require-namespaces short))
> >              (list short (list long))))))))
> >
> > +(defun cljr--in-sans-ns-keyword-p ()
>
> ok I rather prefer the -p style preds. But really don't want to go down
> the rabbit hole of renaming all our foo? style preds in this PR.
>
>    - will create a separate PR replacing all our own ? predicates with -p
>    ones
>    - keep this fn -p style
>
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub
> <https://github.com/clojure-emacs/clj-refactor.el/pull/311/files/5f2adca5f6175d34508f83ea951cb48356fbb4c0#r57476657>
>



-- 
Best Regards,
Bozhidar Batsov

http://www.batsov.com